.github/workflows/ci_action_melodic.yml

```diff
@@ -0,0 +1,30 @@
+name: CI-Melodic
```

Comment:
Why do we need two seperate workflow files? What is the advantage?

Reply:
I think it is just a matter of taste. Correct, @SansoneG ?
I like it, because it is less confusing.

Reply:
The primary reason was to be able to generate CI-Badges for melodic and noetic separately.
Otherwise it is a matter of preference as @martiniil mentioned.